code/solr/Solr5Service.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+class Solr5Service_Core extends Solr4Service_Core
+{
+
+}
+
+class Solr5Service extends Solr4Service
```

Comment:
Could you please separate this into its own file?

Reply:
Sure, I've split versions 3 and 4 out into their own files as well. I've also changed the target